app/controllers/subscriptions_controller.rb

```diff
@@ -33,9 +33,7 @@ def verify
     return frequency_form_redirect unless valid_frequency
 
     if @address.present?
-      @title = GdsApi.email_alert_api
-        .get_subscriber_list(slug: @topic_id)
-        .to_h.dig("subscriber_list", "title")
+      @title = @subscriber_list["title"]
```

Comment:
The commit doesn't seem to mention why this is moved outside of assign_attributes so I was wondering the reason for that?

Is this the only place we use `@title` - might it be simpler to reference `@subscriber_list["title"]` in the view?

Reply:
Sure, fair enough. I've moved it to the view.